contrib/completion/bash-completion.sh

```
@@ -61,10 +61,15 @@ function _resticprofile() {
       ;;
     esac
   fi
+
+  if [[ ! "${COMPREPLY[@]}" == *. ]]; then
```

Shellcheck

[error] Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).

(SC2199)